chain/mainnet/mezo_31612-1/genesis.json

```diff
@@ -1 +1,296 @@
-[]
+{
+  "app_name": "mezod",
+  "app_version": "1.0.0-rc2",
+  "genesis_time": "2025-04-14T08:50:53.767481Z",
+  "chain_id": "mezo_31612-1",
+  "initial_height": 1,
+  "app_hash": null,
+  "app_state": {
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo188vx087k295fxdlx7f4wkznmghajnf0maysxpe",
+            "pub_key": null,
+            "account_number": "0",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo1qcjuy52698gf7xaymfdg00guh3nnse6chm0s2t",
+            "pub_key": null,
+            "account_number": "1",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo14ugypmyfn2whx4dhsw4jwflk2x5fx2vkvmjrzk",
+            "pub_key": null,
+            "account_number": "2",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo1lgrfdvcj38gwcwr0qv4ytdfcygc0mg6slsj3yt",
+            "pub_key": null,
+            "account_number": "3",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        }
+      ]
+    },
+    "authz": {
+      "authorization": []
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
```

Comment:
We may need to separate MEZO minting from genesis. This is a new information from yesterday and we should know it by Wednesday EOD latest. I will try to get the update today.

Reply:
This affects the whole pre-mainnet plan if so. Just flagging.